site/src/sphinx/server-decorator.rst

```diff
@@ -217,6 +217,82 @@ Therefore, you need to specify the decorators as extra parameters:
 A good real-world example of :api:`ServiceWithRoutes` is :api:`GrpcService`.
 See :ref:`server-grpc-decorator` for more information.
 
+Decorating multiple services by path mapping
+--------------------------------------------
+
+If you want to decorate multiple :apiplural:`Service` by path mapping or router matching, you can specify a way to select
+the services which are being decorated with ``decoratorUnder(pathPrefix, ...)`` or ``decorator(Route, ...)``.
```

Comment:
How about just `you can specify decorators using decoratorUnder(...`

Reply:
It looks simple & nice :-)